src/common/components/CoreLayout/Sidebar/types.ts

```diff
@@ -0,0 +1,7 @@
+export type SidebarListItem = {
+  label: string;
+  icon: React.ReactNode;
+  href: string;
+  active?: boolean;
+  external?: boolean;
+};
```

Comment:
Lets move this to `SidebarItem` component instead. It can be exported from there